hnn_core/param/N20.json

```diff
@@ -1,5 +1,5 @@
 {
-	"tstop": 120,
+	"tstop": 170,

```

Comment:
I know it's not ideal to make the simulations we run in our examples longer, however, this is to maintain consistency with `default.json` and the work that will (hopefully) get published soon where we explicitly model the median nerve evoked response.